roles/ffmpeg/defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-ffmpeg_path: /opt/install/ffmpeg_sources
\ No newline at end of file
+ffmpeg_path: "{{ project_base }}/install/ffmpeg_sources"
```

Comment:
I'd like to see the install_path variable here, so if a user overrides it, the ffmpeg_sources directory will end up in the same location as everything else.